packages/clarity-js/src/data/metadata.ts

```diff
@@ -147,9 +150,23 @@ function num(string: string, base: number = 10): number {
 }
 
 function user(): string {
-  let id = getCookie(Constant.CookieKey);
-  // Splitting and looking up first part for forward compatibility, in case we wish to store additional information in a cookie
-  return id && id.length > 0 ? id.split(Constant.Pipe)[0] : shortid(); 
+  let cookie = getCookie(Constant.CookieKey);
+  if(cookie && cookie.length > 0) {
+    // Splitting and looking up first part for forward compatibility, in case we wish to store additional information in a cookie
+    let parts = cookie.split(Constant.Pipe);
+    let userId = cookie.split(Constant.Pipe)[0];
+    if (parts.length === 2) { return userId; } else {
```

Comment:
Minor suggestion: make this >= 2 for forward compat in case we ever add soething else.

Reply:
Good point, done.